src/pylorax/api/v0.py

```diff
@@ -401,8 +401,8 @@ def v0_blueprints_new():
             blueprint = recipe_from_toml(request.data)
         else:
             blueprint = recipe_from_dict(request.get_json(cache=False))
-
-        if VALID_API_STRING.match(blueprint["name"]) is None:
+        
```

Comment:
looks like this added some extra whitespace.

Reply:
Thanks for pointing this out. The whitespace should be removed now.